src/main/resources/application.intg.conf

```diff
@@ -0,0 +1,15 @@
+include "application.conf"
+
+gov_uk_notify {
+  endpoint = "https://api.notifications.service.gov.uk"
+  api_key = "arbitrary_placeholder"
+  api_key = ${?GOV_UK_NOTIFY_API_KEY}
+  on = false
+  on = ${?SEND_GOV_UK_NOTIFICATIONS}
+  transfer_complete_template_id = "arbitrary_placeholder"
+  transfer_complete_template_id = "6fe48fcd-24bf-4a17-b4fe-d121b12a1b4d"
+  metadata_review_template_id = "arbitrary_placeholder"
```

Comment:
Can probably remove these `arbitrary_placeholder` fallbacks now if we're hardcoding anyway?

Reply:
We can remove it but it would mean we have to export these environment variables every time we run locally unless that's okay?